src/main.py

```diff
@@ -1,92 +1,81 @@
 from flask import Flask, render_template, abort
 from flask_talisman import Talisman
 from csp import csp
+import logging
```

Comment:
Unused import

Reply:
Is it not used on line 81? Pycharm was complaining, which is why I assumed it was necessary, but it may just have been an overeager IDE warning. Happy to remove it though!

```
80:  def server_error(e):
81:      logging.exception('An error occurred during a request.')
```

Reply:
Oh you're right! Disregard.